austraits_database.qmd

```diff
@@ -16,3 +16,53 @@ As of October, 2023, AusTraits has:
 
 -   30,000 Australian plant taxa 
 
+## Accessing AusTraits data
+
+There are multiple ways to access and manipulate AusTraits data.\
+
+-   Download the dataset from [Zenodo](https://doi.org/10.5281/zenodo.3568417)\
+
+-   Source AusTraits using the [austraits R package](https://github.com/traitecoevo/austraits)\
+
+-   Clone the austraits.build GitHub repository and build AusTraits from the current [develop branch](https://github.com/traitecoevo/austraits.build) (*not easily reproducible*)
+
+### Download database with austraits.R package
+
+First install and source the package: 
+
+```{r, eval = FALSE}
+install.packages("remotes")  # if it isn't already installed on your machine
+
+remotes::install_github("traitecoevo/austraits", dependencies = TRUE, upgrade = "ask")
+
+library(austraits) 
+```
+
+Before loading AusTraits, see what versions are available,
+
+```{r  eval = FALSE}
+get_versions()
+```
+
+yields:
+
+```{r, eval = FALSE}
+# A tibble: 6 × 3
```

Comment:
It would be preferable if this code was actually run during book build. Let's leave as is for now, but review later after this week's tutorial. Can you make an issue?

Reply:
also for `spatial_data_example.qmd`